pkg/review/commit_review_status_test.go

```diff
@@ -533,6 +533,108 @@ WHERE
 	}
 }
 
+func TestGetPullRequest(t *testing.T) {
+	t.Parallel()
+	cases := []struct {
+		name         string
+		pullRequests []*PullRequest
+		want         *PullRequest
+	}{
+		{
+			name: "returns_first_approving_pull_request",
+			pullRequests: []*PullRequest{
+				{
+					DatabaseID:     1,
+					Number:         23,
+					ReviewDecision: "APPROVED",
+				},
+				{
+					DatabaseID:     2,
+					Number:         24,
+					ReviewDecision: "APPROVED",
+				},
+				{
+					DatabaseID:     5,
+					Number:         345,
+					ReviewDecision: "REVIEW_REQUIRED",
+				},
+			},
+			want: &PullRequest{
+				DatabaseID:     1,
+				Number:         23,
+				ReviewDecision: "APPROVED",
+			},
+		},
+		{
+			name: "returns_nil_when_no_approving_pull_requests",
+			pullRequests: []*PullRequest{
+				{
+					DatabaseID:     1,
+					Number:         23,
+					ReviewDecision: "REVIEW_REQUIRED",
+				},
+				{
+					DatabaseID:     2,
+					Number:         24,
+					ReviewDecision: "REVIEW_REQUIRED",
+				},
+				{
+					DatabaseID:     5,
+					Number:         345,
+					ReviewDecision: "REVIEW_REQUIRED",
+				},
+			},
+			want: nil,
+		},
+		{
+			name:         "returns_nil_when_no_pull_requests",
+			pullRequests: []*PullRequest{},
+			want:         nil,
+		},
+	}
+	for _, tc := range cases {
+		tc := tc
+
+		t.Run(tc.name, func(t *testing.T) {
+			t.Parallel()
+			got := getApprovingPullRequest(tc.pullRequests)
+			if diff := cmp.Diff(got, tc.want); diff != "" {
+				t.Errorf("getCommitHTMLURL unexpected result (-got,+want):\n%s", diff)
```

Comment:
This might be a copy-paste oversight, getCommitHTMLURL -> getApprovingPullRequest

Reply:
fixed.